rules/windows/command_and_control_remote_file_copy_mpcmdrun.toml

```diff
@@ -0,0 +1,45 @@
+[metadata]
+creation_date = "2020/09/03"
+ecs_version = ["1.6.0"]
+maturity = "production"
+updated_date = "2020/09/03"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies MpCmdRun.exe Windows Defender configuration utility being used to download a remote file.
+"""
+from = "now-9m"
+index = ["winlogbeat-*", "logs-endpoint.events.*"]
+language = "kuery"
+license = "Elastic License"
+name = "Remote File Download via MpCmdRun"
+note = """### Investigating Remote File Download via MpCmdRun
+Verify details such as parent process, url reputation and downloaded file details, in addition to that in case of download errors MpCmdRun logs this information automatically in the Appdata Temp folder in MpCmdRun.log where the download url is recorded."""
+references = ["https://twitter.com/mohammadaskar2/status/1301263551638761477"]
+risk_score = 47
+rule_id = "c6453e73-90eb-4fe7-a98c-cde7bbfc504a"
+severity = "medium"
+tags = ["Elastic", "Windows"]
+type = "query"
+
+query = '''
+event.category:process and not event.type:end and
```

Comment:
why this vs `event.type:start`?

Reply:
changed to event.type:(start or process_started)